pkg/apply/task/apply_task.go

```diff
@@ -213,8 +214,7 @@ func newApplyOptions(taskName string, eventChannel chan<- event.Event, serverSid
 			ch:        eventChannel,
 			groupName: taskName,
 		}).toPrinterFunc(),
-		DynamicClient:  dynamicClient,
-		DryRunVerifier: resource.NewQueryParamVerifier(dynamicClient, openAPIGetter, resource.QueryParamDryRun),
```

Comment:
is this QueryParamVerifier not used any more?

Reply:
yes, as stated in the description:

> With v0.27, DryRunVerifier has been deprecated (https://github.com/kubernetes/kubernetes/pull/114294) without any replacement. Therefore this PR also removes any references.